src/github/pullRequest.ts

```diff
@@ -102,4 +104,46 @@ export class PullRequestApi {
   getAuthor(): string {
     return this.pr.user.login;
   }
+
+  async generateCheckRun(checkResult: CheckData): Promise<void> {
```

Comment:
could you please add some description here of what's expected to happen? as the method name is a bit too broad to understand the end result of it